src/MainWindow.vala

```diff
@@ -102,6 +107,27 @@ public class Installer.MainWindow : Hdy.Window {
         battery_infobar.response.connect (() => {
             battery_infobar.revealed = false;
         });
+
+        var mediakeys_settings = new Settings ("org.gnome.settings-daemon.plugins.media-keys");
+
+        orca_timeout_id = Timeout.add_seconds (3, () => {
```

Comment:
You said 10 secs in the description but it is 3 here

Reply:
Whoops. Thank you lol